tests/unit/test_index.py

```diff
@@ -1,437 +1,464 @@
 import pandas as pd
-import numpy as np
 import pytest
 import warnings
 
-from pinecone.core.client.api_client import Endpoint
```

Comment:
This export is not available, and most substantive changes in this test file are to refactor away from depending on it.

Reply:
Good call, thank you.